redirect.config

```diff
@@ -9,6 +9,9 @@
 
 <add key="/reporting/installation-upgrading-newer-version" value = "/reporting/upgrade/overview"/>
 
+<add key="/reporting/getting-started/first-steps" value = "https://docs.telerik.com/reporting/getting-started/first-steps-designing"/>

```

Comment:
Consider relative URI also for the 'value':
<add key="/reporting/getting-started/first-steps" value = "/reporting/getting-started/first-steps-designing"/>

Reply:
Updated.